index.js

```diff
@@ -217,8 +217,8 @@ function homePageFunctions() {
     });
 
     contributeButton.addEventListener('click', function() {
-        window.location.href = 'contribute.html'; // Navigate to contributionForm.html
-    });
+    const dataLink = contributeButton.getAttribute('data-link');
+    window.location.href = dataLink;
```

Comment:
- [ ] Missing close parentheses for the anonymous function block.
- [ ] Test to ensure it works.